### PR TITLE
cnv: note about CD limit due to layer size

### PR DESCRIPTION
**Pull request by fabiand**
For quay this is mentioned in
https://access.redhat.com/documentation/en-us/red_hat_quay/3.5/html-single/manage_red_hat_quay/index#quay-schema
https://github.com/quay/quay/blob/master/config.py#L594

### CODE REVIEW

**Comment by fabiand** [comment]
@aburdenthehand can you help to shape this PR up?

**Comment by fabiand** [comment]
cc @dankenigsberg

**Comment by aburdenthehand** [comment]
@fabiand I'm not really working on cnv docs anymore but I'll ping the channel and see if I can get eyes on it. 
Otherwise, I can take a look next week.

**Comment by fabiand** [comment]
Ack, thanks!

**Comment by fabiand** [comment]
@bgaydosrh something you can potentially helpw ith?

**Comment by bgaydosrh** [comment]
> @bgaydosrh something you can potentially helpw ith?

Hi @fabiand - if all you need to is a merge and pick for this content, I can do that. Let me know and I'll be glad to do it if you approve. Bob

**Comment by fabiand** [comment]
Bob, sorry, I have missed your comment!

I do think that my wording might need an update, and there is also a link, it would be good if we can turn this into a module link (if that is the right temr)

File: modules/virt-preparing-container-disk-for-vms.adoc [hunk 1]
```diff
@@ -7,6 +7,19 @@
...
+[NOTE]
+====
+The size of a disk image inside a containerDisk is limited by the size of the maxium layer size of any registry where the containerDisk will be hosted.
+For Red Hat Quay the default limit is 20GiB and can be configured using the `MAXIMUM_LAYER_SIZE` in the https://access.redhat.com/documentation/en-us/red_hat_quay/3.5/html-single/manage_red_hat_quay/index#quay-schema[Red Hat Quay configuration].
+====
+
+[IMPORTANT]
+====
+Large container images - and thus containerDisks - can lead to disk pressure on worker nodes and in the worst case lead to node unavailability.
+NEEDS reference
```
Comment (fabiand) [inline modules/virt-preparing-container-disk-for-vms.adoc:19]: We could link to https://access.redhat.com/solutions/3396081

Comment (bgaydosrh) [reply modules/virt-preparing-container-disk-for-vms.adoc:19]: Thanks @fabiand - Yes we can use a LINK to that KB as long as its a verified solution.

**Comment by aspauldi** [comment]
@fabiand , hello! Are you able to create a bug or story so that we can track this work? Thanks!

**Comment by bgaydosrh** [comment]
See PR: https://github.com/openshift/openshift-docs/pull/38474 

@fabiand or @ctomasko - Please feel free to close this PR (https://github.com/openshift/openshift-docs/pull/32695) as https://github.com/openshift/openshift-docs/pull/38474 replaces it. I do not have authorization to close PRs. Thanks.

**Comment by fabiand** [comment]
/close

in favor of #38474

**Pull request closed without merging**